ofl/loversquarrel/DESCRIPTION.en_us.html

```diff
@@ -1 +1,10 @@
-<p>Lovers Quarrel is an ornate calligraphic script. It has beautifully embellished uppercase characters and clean, legible lowercase forms.</p> <p>Lovers Quarrel is a Unicode typeface that supports languages that use the Latin script and its variants, and could be expanded to support other scripts. To contribute to the project contact <a href="mailto:typesetit@att.net">Rob Leuschke</a>.</p>
\ No newline at end of file
+<p>
+MonteCarlo is a beautiful formal script— both contemporary and traditional. This connecting script’s italic is slight, making it an extremely legible design.
```

Comment:
I think you accidentally copied the wrong description

Reply:
No, I forgot to save and push the latest changes.